lib/codit/population/networks/home.py

```diff
@@ -0,0 +1,113 @@
+"""
+An spatial attribute for each household
+"""
+
+import pandas as pd
+import os
+import csv
+import numpy as np
+from codit import share_dir
+
+DATA_PATH = os.path.join(share_dir(), 'codit', 'data')
+COORDINATES_CSV = os.path.join(DATA_PATH, 'city', 'population', 'coordinates_leeds.csv')
+TYPES_CONSTRAINTS_CSV = os.path.join(DATA_PATH, 'city', 'population', 'types_households_constraints_leeds.csv')
```

Comment:
Don't want to hardcode anything specifically about a given city, e.g. about Leeds